module-1/README.md

```diff
@@ -69,67 +70,101 @@ In the terminal, change directory to the newly cloned repository directory:
+To **open a file** in Cloud9, use the File Explorer on the left panel and double click `bucket-policy.json`:
 
-![bucket-policy-image.png](/images/module-1/bucket-policy-image.png)
+![bucket-policy-image.png](/images/module-1/bucket-policy.png)
 
-This will open `bucket-policy.json` in the File Editor panel.  Replace the string shown with your chosen bucket name used in the previous commands:
+This will open `bucket-policy.json` in the File Editor panel.  Replace the string shown on both lines 9 and 12 with your Id and chosen bucket name used in the previous commands:
 
 ![replace-bucket-name.png](/images/module-1/replace-bucket-name.png)
 
+In Cloud9, open the `website-cloudfront-distribution.json` file and edit the bucket name on line 12 and the Id on line 14:
 
-Execute the following CLI command to add a public bucket policy to your website:
+![bucket-policy-image.png](/images/module-1/cloudfront-distribution.png)
+
+Execute the following CLI command to add a public bucket policy to allow CloudFront:
 
 ```
 aws s3api put-bucket-policy --bucket REPLACE_ME_BUCKET_NAME --policy file://~/environment/aws-modern-application-workshop/module-1/aws-cli/website-bucket-policy.json
 ```
 
 #### Publish the Website Content to S3
 
-Now that our new website bucket is configured appropriately, let's add the first iteration of the Mythical Mysfits homepage to the bucket.  Use the following S3 CLI command that mimics the linux command for copying files (**cp**) to copy the provided index.html page locally from your IDE up to the new S3 bucket (replacing the bucket name appropriately).
+Now that our S3 bucket is created, let's add the first iteration of the Mythical Mysfits homepage to the bucket.  Use the following S3 CLI command that mimics the linux command for copying files (**cp**) to copy the provided index.html page locally from your IDE up to the new S3 bucket (replacing the bucket name appropriately).
 
 ```
 aws s3 cp ~/environment/aws-modern-application-workshop/module-1/web/index.html s3://REPLACE_ME_BUCKET_NAME/index.html
 ```
 
-Now, open up your favorite web browser and enter one of the below URIs into the address bar.  One of the below URIs contains a '.' before the region name, and the other a '-'. Which you should use depends on the region you're using.
+#### Create CloudFront Distribution
 
-The string to replace **REPLACE_ME_YOUR_REGION** should match whichever region you created the S3 bucket within (eg: us-east-1):
+Using the AWS CLI, we will create a CloudFront distribution, and configure the S3 bucket as the origin. The JSON document for the necessary CloudFront configuration is located at: `~/environment/aws-modern-application-workshop/module-1/aws-cli/website-cloudfront-distribution.json`.  This file contains two strings that need to be replaced, the Id (indicated with `REPLACE_ME_ID`), and the bucket name you've chosen (indicated with `REPLACE_ME_BUCKET_NAME`).  
```

Comment:
Another REPLACE_ME_ID that should be changed to REPLACE_ME_CLOUDFRONT_ORIGIN_ACCESS_IDENTITY

Reply:
Replaced